composer.json

```diff
@@ -20,7 +20,9 @@
         "cakephp/cakephp": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "<6.0"
+        "phpunit/phpunit": "<6.0",
+        "squizlabs/php_codesniffer": "^3.2",
```

Comment:
Explicitly adding `squizlabs/php_codesniffer` is not required since it's already a dependency of `cakephp/cakephp-codesniffer`.